README.md

```diff
@@ -46,3 +46,65 @@ this project took
 * advanced option handling
 * `%autosetup`/`%autopatch`
 * many more nuts and bolts under the hood
+
+## how to install
+
+debbuild's packages are (naturally) built by debbuild itself. You can generate an installable
+`.deb` file for your distribution by running the `./bootstrap.sh` script.
```

Comment:
We also release installable debs for every release too.

Reply:
That is true, but the [current tag](https://github.com/debbuild/debbuild/releases/tag/21.01.0) only features Ubuntu 20.04 debs. While they would probably install on buster just fine, it feels safer to build the deb directly for the targeted distro.

Should I add a note about the prebuilt debs being available?

Are you planning to offer a repo with [GitHub Packages](https://github.com/features/packages)?

Reply:
We offer them for Debian and Ubuntu on the openSUSE Build Service:

* Debian: https://download.opensuse.org/repositories/Debian:/debbuild/
* Ubuntu: https://download.opensuse.org/repositories/Ubuntu:/debbuild/

Reply:
Sounds good, I [added a note about those to the README](https://github.com/debbuild/debbuild/pull/210/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R55-R60).